hknweb/candidate/templates/candidate/candreq.html

```diff
@@ -38,9 +38,9 @@ <h3>Request Officer Challenge Confirmation</h3>
                 {% if field.label == 'Proof' %}
                 <br/>
                 <span class="blue-note">
-                    You can optionally include some proof that you've completed
-                    the challenge. If your proof is a file, you can send it to
-                    the Slack channel, or upload to Google Drive and paste a link here.
+                    You can include some proof that you've completed the challenge.
+                    If your proof is a file, you can send it to #challenges on Slack
+                    or upload to Google Drive. If you do so, please paste a link here.
```

Comment:
Can you change this note to [this one](https://github.com/compserv/hknweb/pull/190/files?file-filters%5B%5D=.css&file-filters%5B%5D=.html&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-e0e7fdcda28ecef969c2fbade9816158R39) so that two notes match up since they say pretty much the same thing? Also, is it an important point that this officer challenge one can be a file, not just an image just the BitByteActivity proof?

Reply:
I think they're different? Officer challenges can be anything

Reply:
I meant the instructions 

> You can obtain the link by clicking "More actions->Copy link" on Slack. The link should start with "https://hkn.slack.com/archives/"

Additionally, Danny would prefer if the proof instruction were above the proof and if the text color was gray?

Reply:
*were